jest.config.js

```diff
@@ -0,0 +1,33 @@
+// Import WP-scripts presets to extend them,
+// see https://developer.wordpress.org/block-editor/packages/packages-scripts/#advanced-information-11.
+const defaultConfig = require( '@wordpress/scripts/config/jest-unit.config' );
+
+const wcPackagesNeedTransform = [
+	'components',
+	'currency',
+	'data',
+	'date',
+	'navigation',
+	'number',
+	'tracks',
+	'experimental',
+].join( '|' );
+
+module.exports = {
+	...defaultConfig,
+	// Workaround https://github.com/woocommerce/woocommerce-admin/issues/6483.
+	transformIgnorePatterns: [
+		`<rootDir>/node_modules/(?!@woocommerce/(${ wcPackagesNeedTransform })(/node_modules/@woocommerce/(${ wcPackagesNeedTransform }))?/build/)`,
+	],
```

Comment:
I'm not sure, but I think we may not need that here, as we externalize the woo dependencies here (as we use `@woocommerce/DEWP` hear, as opposed to `@wordpress/DEWP` in GLA), and if possible, I'd avoid this bit, as it's very sketchy and fragile.

Reply:
Thanks for pointing this out @tomalec 

Not sure if I understand it tho, does it mean that because @woocommerce/DEWP it works differently. What is exactly the difference? Trying to understand the context here.

Reply:
My bad, tested it locally, and those lines are still needed.

Because DEWP might work differently on E2E, but it shouldn't change a thing for unit tests.